challenge-quizzes/built_ins/exercise8_15/readme.md

```diff
@@ -0,0 +1,37 @@
+## Lesson 8 Exercise 15
+```script.js
```

Comment:
[optional] introductory text can directly be written in Markdown rather than comments in a codeblock.